home-page.html

```diff
@@ -63,80 +116,26 @@ <h2>INHERITANCE</h2>
             </div>
         </div>
         <div class="big-orange-button">
-            <a href="">Join Today!!!</a>
+            <a href="">Join CC GLOBAL-SUMMIT 2015</a>
```

Comment:
- [ ] Please use a button here to fix the design. This will make your design match the provided design.

**Your project**
![image](https://user-images.githubusercontent.com/74411135/235261962-58ad5ba1-f9d6-40ed-9f6d-2043763ddba8.png)

**Provided design**
![image](https://user-images.githubusercontent.com/74411135/235262039-28c22436-3b23-4b11-b7f0-076d4b1efa76.png)

This will make your project way prettier which will improve the user experience 😄